### PR TITLE
🦥 Disable icon jumping when a bell is received

### PR DESCRIPTION
**Pull request by boxmein**
This PR is essentially a manual fix for #2492 and #2950. It changes the terminal bell event handler so that it no longer sets the `"urgent"` flag on the window. macOS uses the urgent flag to detect if the dock icon should be jumping.

As discussed in #2950, alacritty will expect the inner process to set a terminal mode flag instead. However, in my experience, it is almost impossible to find what is sending the bell. For me, the optimal fix was just reacting to bells normally *without* causing the dock icon to jump.

I assume the PR isn't really a candidate for release - but it may help someone get rid of an annoying icon jump issue.

I track alacritty in my forked repository and add this commit on top, so that there's a version of alacritty without the dock icon jumping functionality.

### CODE REVIEW

**Comment by kchibisov** [comment]
But you could set the mode to disable jumping from the `shell` launch/propmt and you'll never see it jumping regardless of what you run?

**Comment by kchibisov** [comment]
Yes, the mentioned escape sequence.

**Comment by kchibisov** [comment]
Your terminal here is tmux, you could disable bell in it.

**Comment by boxmein** [comment]
It already is disabled :) my config file includes these lines:

```
set-option -g bell-action none
set-option -g visual-bell off
```

**Comment by mrded** [comment]
TL;DR: https://github.com/alacritty/alacritty/pull/4304
```
$ printf "\e[?1042l"
```
